<launch>
	<node name = "controller_worm_node"
    		pkg = "ros_robotics"
    		type = "controller_worm"
    		output = "screen"/>
</launch>
